### ISSUE TITLE
No code being generated  to sign in/activate

### ISSUE THREAD

**Issue by jlitz** [yuliskov/SmartTubeNext#515]
just reinstalled   cannot sign in as no code is being generated

**Comment by laobibibibibibi** [comment]
Yes, the user code cannot be displayed after reinstalling in the morning, and the account cannot be displayed on another TV without reinstalling

**Comment by caspian80** [comment]
My account suddenly signed out for no reason and this issue is also happening to me. No code is being generated to sign in.

**Comment by laobibibibibibi** [comment]
ok, has returned to normal

**Comment by ontwikkelfabriek** [comment]
Same thing over here. no code is generated.

**Comment by foreveryoungforty** [comment]
It works perfect in update 10.127 here  - @jlitz please close issue if ok

**Comment by Australian-Infomaniac** [comment]
Same here on Beta installed several months ago. Will try newest.

(While I have the 'microphone:
I want to say I hope the app can be kept as free of ads as possible.
That's the main reason I use it, because YT has become really greedy, selfish and in-your-face with countless ads.
I can actually watch YT on my TV via Chrome on Kodi.
But today I decided to return to YTNext because I got sick of the ridiculous number of ads on the standard YT - just OTT !
I hope this sign-out isn't automatically triggered when there is a new update available)

**Comment by mizzunet** [comment]
Same here on latest beta version.
